docs/docsite/rst/dev_guide/developing_modules_in_bulk.rst

```diff
@@ -0,0 +1,156 @@
+Partner information for submitting modules
+==========================================
+
+.. contents:: Topics
+
+.. _module_dev_welcome:
+
+Welcome
+```````
+This section discusses some of the specifics around getting multiple related modules into Ansible.
+
+The information in the page is useful for both companies wishing to add modules for their own products, as well as for users of 3rd party products wishing to add Ansible support.
+
+It's based on the knowledge on the Core Team and Community on what we've found is the best practice for the module developer(s), the Community, and the Ansible Core Team.
```

Comment:
Tough sentence. Suggest: "It's based on module development best practices that the Ansible core team and community have accumulated."